fe/fe-core/src/main/java/com/starrocks/common/Config.java

```diff
@@ -1507,6 +1507,29 @@ public class Config extends ConfigBase {
     @ConfField(mutable = true)
     public static int quorom_publish_wait_time_ms = 5000;
 
+    /**
+     * FE journal queue size
+     * Write log will fail if queue is full
+     **/
+    @ConfField(mutable = true)
+    public static int metadata_journal_queue_size = 1000;
+
+    /**
+     * The maxium size(key+value) of journal entity to write as a batch
+     * Increase this configuration if journal queue is always full
+     * TODO: set default value
+     **/
+    @ConfField(mutable = true)
+    public static int metadata_journal_max_batch_size_mb = 10;
+
+    /**
+     * The maxium number of journal entity to write as a batch
+     * Increase this configuration if journal queue is always full
+     * TODO: set default value
```

Comment:
what this TODO mean?

Reply:
The default values of these two config need to be evaluated with some performance tests which I will run later. Now I only set them to a larger value to ensure it won't be the bottleneck.